dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -349,6 +349,8 @@ def report(self, remove_disabled_flag: bool = False) -> Dict:
                     continue
                 elif profile_key == "times":
                     continue
+                elif profile_key == "psi":
```

Comment:
add option for disabling flag `psi`

Reply:
can we combine this with `times` and create a `in [...]`?

Reply:
we could... I kinda see `times` and `psi` as two distinct keys ... whereas `stddev` and `variance` together make sense becuase they are derivatives of each other

Reply:
they are different, but the logic is the same. if the key is this, continue. whereas stddev / variance are dependent hence, if one is not there, we have to pop the other.

Reply:
update